app/templates/account/israel-signup-view.jade

```diff
@@ -0,0 +1,45 @@
+extends /templates/base-flat
+
+block content
+  
+  .container
+    h1.m-t-3.m-b-2.text-center Create Account
+    
+    if loading
+      h5.text-center Loading
+    
+    else if fatalError
+      .alert.alert-danger.text-center
+  
+        if fatalError === 'signed-in'
+          h3 You already have an account
+          a(href="/play") Play CodeCombat!
+          
+        else if fatalError === 'missing-input'
+          h3 Not enough information was provided
+          p Need at least "il_id" and "email" included in GET parameters.
+          
+        else if fatalError === 'email-exists'
+          h3 Your email is already in use
+          a.login-button(data-i18n="login.log_in")
+  
+        else
+          h3 Error
+          p= fatalError
+          
+    else
+      .row
+        .col-md-6.col-md-offset-3.col-sm-8.col-sm-offset-2
+          form
+            .form-group
+              label.control-label(for="username-input" data-i18n="general.username")
+              input.form-control.input-lg(name="name" value=name)
+            
+            .form-group
+              label.control-label(for="password-input" data-i18n="general.password")
+              input.form-control.input-lg(name="password" type="password" value=password)
+              
+            if formError
+              .alert.alert-danger.text-center= formError
+
+            button#create-account-btn.btn.btn-lg.btn-block.btn-navy(type='submit' data-i18n="login.sign_up")
```

Comment:
Our (intended, but not very thoroughly followed) internal style guidelines use double-quotes in jade/scss and single-quotes in javascript/coffeescript (including JS inside jade).